audits/multicall/202309-threat-model-multicaller/202309-threat-model-multicaller.md

```diff
@@ -0,0 +1,78 @@
+## Background
```

Comment:
This .md file should be in ./audits/multicall/

Reply:
done